iothub_client/tests/common_longhaul/iothub_client_common_longhaul.c

```diff
@@ -1675,11 +1675,20 @@ static void on_twin_report_state_completed(int status_code, void* userContextCal
 
 static void check_for_reported_properties_update_on_service_side(IOTHUB_LONGHAUL_RESOURCES* iotHubLonghaul)
 {
-    char* twin;
+    char* twin = NULL;
 
-    if ((twin = IoTHubDeviceTwin_GetTwin(
-        iotHubLonghaul->iotHubSvcDevTwinHandle,
-        iotHubLonghaul->deviceInfo->deviceId)) == NULL)
+    for (int i = 0; i < 3 && twin == NULL; i++)
```

Comment:
Super minor comment: define a macro like "#define MAX_TWIN_GET_ATTEMPT_COUNT 3" to make it a little more explicit why "3".